osu.Game/Overlays/BeatmapSet/Buttons/PreviewButton.cs

```diff
@@ -24,16 +24,14 @@ public partial class PreviewButton : OsuClickableContainer
 
         public IBindable<bool> Playing => playButton.Playing;
 
-        public APIBeatmapSet BeatmapSet
+        public IBeatmapSetInfo BeatmapSet
```

Comment:
As above, why?

It looks as if you tried to use this first to implement the feature and then settled on `BeatmapCardThumbnail` because that didn't work but you didn't revert all of these changes?

Reply:
Didn't revert because no reason to. Can if you want though 🤷